modules/service-fips.sh

```diff
@@ -50,6 +54,63 @@ fips_disable() {
     not_supported 'Disabling FIPS'
 }
 
+fips_updates_enable() {
+    local token="$1"
+    local bypass_prompt="$2"
+    local fips_configured=0
+    local fips_kernel_version=0
+
+    check_token "$FIPS_UPDATES_REPO_URL" "$token"
+
+    echo "Updating FIPS packages will take the system out of FIPS compliance."
```

Comment:
Wording suggestion. Instead of `Updating FIPS packages will take the system out of FIPS compliance`, use something like `Enabling the fips-updates repository will take the system out of FIPS compliance`. The problem isn't updating the fips packages, but updating from fips-updates, right? Essentially we have two channels:
 * fips: for the **certified** packages
 * fips-updates: for these **uncertified** updates

There is nothing preventing `fips` from eventually receiving a new set of certified packages, in which case updating the packages should be just fine.

That being said, I don't want to bikeshed, whichever way you prefer is fine :)

Reply:
Only updates from fips-updates ppa is uncertified. Enabling fips-updates repository by itself will not make the system non-compliant, but installing updates from that repo will. So may be I can change it to "Installing updates from fips-updates repository will take the system out of FIPS compliance". Do you agree?

Reply:
yep

Reply:
fixed both formatting with status and the user prompt message.